airflow/cli/commands/remote_commands/dag_command.py

```diff
@@ -224,8 +224,8 @@ def _get_dagbag_dag_details(dag: DAG) -> dict:
     return {
         "dag_id": dag.dag_id,
         "dag_display_name": dag.dag_display_name,
-        "is_paused": dag.get_is_paused(),
-        "is_active": dag.get_is_active(),
+        "is_paused": None,
+        "is_active": None,
```

Comment:
This code is only reachable when a DAG has no corresponding DagModel, in which case the original implementation would always return None anyway.